itests/src/test/java/org/apache/unomi/itests/SegmentIT.java

```diff
@@ -46,8 +47,15 @@
     @Filter(timeout = 600000)
     protected SegmentService segmentService;
 
+    @Inject @Filter(timeout = 600000)
+    protected BundleWatcher bundleWatcher;
+
     @Before
     public void setUp() throws InterruptedException {
+        while (!bundleWatcher.isStartupComplete()) {
```

Comment:
Ideally we should move this setup to BaseIT since all the tests should have it, but we should do this in a separate PR.